tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -70,7 +70,13 @@ GOOGLE_RTC_AEC_SUPPORT=1,DEEP_BUF_SPK=true,PLAYBACK_PIPELINE_SRC=dts"
 #ES83x6+HDMI-IN
 "cavs-es83x6\;sof-mtl-es83x6-ssp1-hdmi-ssp02\;PLATFORM=mtl,PREPROCESS_PLUGINS=nhlt,\
 NHLT_BIN=nhlt-sof-mtl-es83x6-ssp1-hdmi-ssp02.bin,HEADSET_SSP_DAI_INDEX=1,\
-HEADSET_CODEC_NAME=SSP1-Codec,HDMI_IN_CAPTURE=true"
+HEADSET_CODEC=true,HEADSET_CODEC_NAME=SSP1-Codec,HDMI1_ID=3,HDMI2_ID=4,HDMI3_ID=5,\
+HDMI_IN_CAPTURE=true"
+
+#No HeadsetCodec+HDMI-IN
+"cavs-es83x6\;sof-mtl-hdmi-ssp02\;PLATFORM=mtl,PREPROCESS_PLUGINS=nhlt,\
+NHLT_BIN=nhlt-sof-mtl-hdmi-ssp02.bin,HEADSET_CODEC=false,HDMI_IN_CAPTURE=true,\
+HDMI_IN_1_ID=0,HDMI_IN_2_ID=1"
```

Comment:
LGTM, but please split the commit to 1. add HEADSET_CODEC macro. 2. Add the sof-mtl-hdmi-ssp02 topology

Reply:
done